presto-native-execution/presto_cpp/main/operators/tests/UnsaferowShuffleTest.cpp

```diff
@@ -233,6 +242,18 @@ class UnsafeRowShuffleTest : public exec::test::OperatorTestBase {
     serializer->deserialize(input.get(), pool(), rowType, &result, nullptr);
     return result;
   }
+
+  RowVectorPtr copyResultVector(RowVectorPtr result) {
+    auto vector = RowVector::createEmpty(result->type(), pool());
+    vector->resize(result->size());
+    for (int c = 0; c < result->children().size(); c++) {
```

Comment:
Is this loop needed? I think you can call vector->copy directly.

Reply:
That was what I was originally expecting. But it seems like a bug in copy. It expects the destination to have the same children by default. That's why I ended doing this.